config/prometheus.php

```diff
@@ -43,4 +43,17 @@
      * Allow storage to be wiped after a render of data in metrics controller.
      */
     'wipe_storage_after_rendering' => false,
+
+    /**
```

Comment:
Could you make this doc block smaller. Only 1 or 2 sentences. Move the rest of the explanation to a new section in the readme.